vignettes/epichains.Rmd

```diff
@@ -0,0 +1,356 @@
+---
```

Comment:
That's interesting. I think it's because `truncdist` is not properly imported. It is currently listed as "Suggests" instead of "Imports", but also not "@importFrom", where it is used. I will fix this in a separate issue.